test/acceptance/features/unbindAppToService.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Unbind an application from a service
+
+    As a user of Service Binding Operator
+    I want to unbind an application from a service
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+
+    @disabled
+    Scenario: Unbind a generic test application from database
+        Given OLM Operator "backend" is running
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: example-backend
+                annotations:
+                    service.binding/host: path={.spec.host}
+                    service.binding/username: path={.spec.username}
+            spec:
+                host: example.com
+                username: foo
+            """
+        * Generic test application "generic-app-a-d-u" is running
+        * Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-a-d-u
+            spec:
+                application:
+                    name: generic-app-a-d-u
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: example-backend
+                    id: backend
+            """
+        * The application env var "BACKEND_HOST" has value "example.com"
+        * The application env var "BACKEND_USERNAME" has value "foo"
+        * Secret "binding-request-a-d-u" contains "BACKEND_HOST" key with value "example.com"
+        * Secret "binding-request-a-d-u" contains "BACKEND_USERNAME" key with value "foo"
```

Comment:
since we checked for env vars, we do not need to check for secrets - the only what counts to users is env var availability

Reply:
Done